pkg/controller/signup_test.go

```diff
@@ -82,8 +88,8 @@ func (s *TestSignupSuite) TestSignupPostHandler() {
 			},
 		}
 
-		svc.MockCreateUserSignup = func(username, userID string) (*crtapi.UserSignup, error) {
-			assert.Equal(s.T(), expectedUserID, userID)
+		svc.MockCreateUserSignup = func(username, userID, email string) (*crtapi.UserSignup, error) {
+			assert.Equal(s.T(), expectedUserID, userID, email)
```

Comment:
?
shouldn't it be something like that instead?
```suggestion
			assert.Equal(s.T(), expectedUserID, userID)
			assert.Equal(s.T(), expectedEmail, email)
```

Reply:
sorry.. refactoring error - fixed here: f7253526905ab1e064b76a0b63b0fe5f316eb2d6